examples/src/bin/whoami.rs

```diff
@@ -1,18 +1,23 @@
-use reqwest::header::{HeaderMap, HeaderValue};
-use serde_json::json;
 use std::env;
 use std::error::Error;
-use tkhq_api_key_stamper::{stamp, TurnkeyApiKey};
-
+use tkhq_api_key_stamper::TurnkeyApiKey;
+use tkhq_client::generated::GetWhoamiRequest;
+use tkhq_client::RetryConfig;
 // See <https://docs.turnkey.com/api-reference/sessions/who-am-i> for documentation
 const TURNKEY_API_HOST: &str = "https://api.turnkey.com";
-const WHOAMI_PATH: &str = "/public/v1/query/whoami";
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error>> {
     // Load .env file from the example folder (examples/.env)
     let current_dir = env::current_dir()?; // should be the workspace root
-    dotenv::from_path(current_dir.join("examples").join(".env"))?;
+    let env_path = current_dir.join("examples").join(".env");
```

Comment:
nit: Could maybe create a function to avoid the repetition for further examples of getting the env file

Reply:
Yep. I think I'll abstract this out. Good idea.